$OpenBSD: patch-includes_definitions_inc_php,v 1.16 2016/05/06 19:31:08 sthen Exp $

Minimal handling for running inside chroot. Not really recommended and some
functionality will be lost, but it might be enough for some users.

--- includes/definitions.inc.php.orig	Tue May  3 12:24:33 2016
+++ includes/definitions.inc.php	Tue May  3 12:24:43 2016
@@ -2109,10 +2109,12 @@ $config['device_types'][$i]['text'] = 'Appliance';
 $config['device_types'][$i]['type'] = 'appliance';
 $config['device_types'][$i]['icon'] = 'appliance.png';
 
+if (!is_dir($config['install_dir']) && is_dir('/var/www' . $config['install_dir'])) { $config['install_dir'] = '/var/www' . $config['install_dir']; }
+
 //
 // No changes below this line #
 //
-$config['version']              = '2015.master';
+$config['version']              = '%LNMS_VERSION%';
 $config['project_name_version'] = $config['project_name'].' '.$config['version'];
 
 if (isset($config['rrdgraph_def_text'])) {
